resources/arduino_files/shared/PinUtils.h

```diff
@@ -9,4 +9,7 @@ bool is_pin_on(int pin_num);
 void toggle_pin(int pin_num);
 void toggle_led();
 
+// Read the mode (INPUT, OUTPUT, or INPUT_PULLUP) for a pin.
+int pinMode(int pin);
```

Comment:
Should the return value go somewhere?

Reply:
I don't understand. Does the comment/function make more sense if I documented it as:
```c++
// Return the mode (INPUT, OUTPUT, or INPUT_PULLUP) for a pin.
```
(i.e. change **Read** to **Return**).